src/app/modules/group/components/pending-request/pending-request.component.spec.ts

```diff
@@ -1,276 +0,0 @@
-import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
You can split the tests between the two components (pending joining requests and pending requests), it should not be that hard. These tests are quite useful as these component are a bit difficult to test.

Reply:
fixed